File: api/Entities/Document.cs
#nullable disable

namespace api.Entities
{
    public partial class Document
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string FilePath { get; set; }
        public string MimeType { get; set; }
        public string UploadedBy { get; set; }
    }
}

File: legacy/app/Document.php
<?php

namespace App;

use Conner\Tagging\Taggable;
use Illuminate\Database\Eloquent\Model;

class Document extends Model
{
	use Taggable;

	/**
	 * Database table used by the model.
	 *
	 * @var string
	 */
	protected $table = 'documents';

	/**
	 * Mass assignable attributes.
	 *
	 * @var array
	 */
	protected $fillable = [
		'name',
		'description',
		'mime_type',
		'uploaded_by'
	];

}
